Arrow Comm Call (#583)
ropensci/roweb3: content/commcalls/2023-06-28/index.md

---
title: "A Journey through Arrow in R"
dateStart: 2023-06-28T16:00:00 # UTC!!
dateEnd: 2023-06-28T17:00:00 # UTC!!
date: 2023-06-28T16:00:00 # UTC!! same as dateEnd
publishDate: 2023-06-28
description: Community Call covering {arrow} intro and use cases.
author:
  - Stephanie Hazlitt
  - Jonathan Keane
  - Nic Crane
organizer: Yanina Bellini Saibene
location: Community call (teleconference)
country: "\U0001F310"
attendees: All are welcome
slug: jun2023-arrow
notes: 
outputs:
- HTML
- Calendar 
# vimeo:
#  - id: ''
resources:
  - url: https://arrow.apache.org/docs/r/index.html
    name: Arrow R Package Documentation  
    details: The arrow R package exposes an interface to the Arrow C++ library, enabling access to many of its features in R. It provides low-level access to the Arrow C++ library API and higher-level access through a dplyr backend and familiar R functions.
 
speakers:  
  - name: Nic Crane
    bio: 
    img: /img/community/nic-crane.png
  - name: Jonathan Keane

>     bio:        

Author: ```suggestion
    bio:  Jonathan Keane is an engineering manager with a background in software engineering and data science. Jonathan is a part of the team who maintains the Arrow project including the Arrow R package.      
```

    img: /img/community/jonathan-keane.png
  - name: Stephanie Hazlitt 

>     bio: 

Author: ```suggestion
    bio: Director, Data Science Partnerships @ BC Stats 🇨🇦
```

    img: /img/community/stephanie-hazlitt.jpeg  
tags:
  - community call
  - events
  - arrow
deets: |
    Meeting ID: 890 7662 9765 
    Passcode: 385358
zoomurl: https://numfocus-org.zoom.us/j/89076629765?pwd=cjRFeHN1TlVXdDFRK0FXRnlzcENuZz09
---

Apache Arrow is a software development platform for building high performance applications that process and transport large data sets. It is designed to improve the performance of data analysis methods, and to increase the efficiency of moving data from one system or programming language to another.

> In this community call moderate by Stephanie Hazlitt our speakers, Nic Crane and Jonathan Keane, will lead us through the Arrow package.

Author: ```suggestion
In this community call moderated by Stephanie Hazlitt, our speakers, Nic Crane and Jonathan Keane, will lead us through the Arrow R package.
```
